File: src/main/java/io/openfuture/openmessanger/service/dto/LoginSmsVerifyRequest.java
package io.openfuture.openmessanger.service.dto;

import javax.validation.constraints.NotBlank;

import lombok.Data;

@Data
public class LoginSmsVerifyRequest {

    @NotBlank
    private String sessionId;

    @NotBlank
    private String username;

    @NotBlank
    private String sms;
}

File: src/main/kotlin/io/openfuture/openmessenger/service/dto/LoginSmsVerifyRequest.kt
package io.openfuture.openmessenger.service.dto

import javax.validation.constraints.NotBlank

data class LoginSmsVerifyRequest(
    val sessionId: @NotBlank String? = null,
    val username: @NotBlank String? = null,
    val sms: @NotBlank String? = null
)